README.md

```diff
@@ -11,6 +11,7 @@ Created with [Python3](https://www.python.org/).
 ## Features
 
 * `wkcuber`: Convert image stacks to fully ready WKW datasets (includes downsampling, compressing and metadata generation)
+* `wkcuber.export_wkw_as_tiff`: Convert WKW datasets to a tiff stack (includes tiling to `z/y/x.tiff` folder structure)
```

Comment:
"includes" could make it sound as if tiff export would always be tiling. I'd write `... (writing as tiles to a ... folder structure is also supported)`